examples/golden_graph/node.cpp

```diff
@@ -1,3 +1,25 @@
 #include "node.hpp"
 
 Node::Node(int id, int value) : m_id {id}, m_value {value} {}
+
+int Node::getID() const
+{
+	return m_id;
+}
+
+int Node::getValue() const
+{
+	return m_value;
+}
+
+std::vector<Edges*>& Node::getEdges() const
+{
+	return m_edges;
+}
+
+void Node::print()
+{
+	std::cout << "\nID: " << getID()
+		  << "\nValue: " << getValue()
+		  << "\nEdges: " << getEdges().print() << std::endl;
```

Comment:
std::vector doesn't have print() function.
Let's print it manually.

Reply:
You are very right, but I did it temporarily and knowing that because the Edge Class is not completely ready, the point is so that all viewers can imagine what will be done in that piece of code.